src/app/modules/card-cluster/card-cluster.component.html

```diff
@@ -1,92 +1,107 @@
-                                    size="small"
-                                    [value]="card.name"
-                                    groupName="cardClusterCheckbox"
-                                    [isChecked]="cardArray.buttonClicked === 'CardCluster'+(x+1)"
-                                    [showTitle]="false"
-                                    [disabled]="card.disabled"
-                                    tabIndex="-1">
-                                </app-choice>
-                            </div>
-                            <div class='ds-l-col' [ngClass]="{'ds-u-padding-x--0': cardArray.showRadioButton}">
-                                <div class="ds-u-font-size--h3 ds-u-font-weight--bold noPoint"
-                                    [ngClass]="card.classValue">
-                                    <fa-icon [icon]="card.valueIcon" *ngIf="card.valueIcon"></fa-icon>
-                                    {{ card.value === undefined || card.value === '' ? '&nbsp;' : card.value }}
-                                </div>
-                                <div class="ds-text noPoint" [ngClass]="card.className">
-                                    <fa-icon [icon]="card.nameIcon" *ngIf="card.nameIcon"></fa-icon>
-                                    {{ !card.name ? '&nbsp;' : card.name }}
-                                </div>
-                            </div>
-                        </div>
-                    </div>
+  <div
+    *ngIf="cardArray.cluster.length"
+    [ngClass]="cardArray.mainCard ? 'ds-l-lg-col--8' : 'ds-l-lg-col--12'"
+    class="ds-l-md-col--12 ds-l-sm-col--12"
+    id="followCard"
+  >
+    <div class="ds-l-row">
+      <ng-container *ngFor="let card of cardArray.cluster; let x = index">
+        <div class="ds-l-col" [ngClass]="(x + 1) % 2 === 0 ? 'padding-L-0' : ''">
+          <div
+            id="{{ 'CardCluster' + (x + 1) }}"
+            [ngClass]="{ active: cardArray.buttonClicked === 'CardCluster' + (x + 1), disabled: card.disabled }"
+            class="{{ 'pointer card cardHeight ds-u-padding--1 ds-u-margin-bottom--2 ' + card.classButton }}"
+            (click)="passAction($event)"
+            (keyup.enter)="passAction($event)"
+            role="button"
+            [attr.tabindex]="card.disabled ? null : 0"
+            [attr.aria-label]="card.ariaLabel || card.name"
+          >
+            <div class="cardClusterContent ds-l-row ds-u-margin-x--0">
+              <div *ngIf="cardArray.showRadioButton" class="ds-l-col--auto ds-u-padding-x--0">
+                <app-choice
+                  [index]="x"
+                  inputType="radio"
+                  size="small"
+                  [value]="card.name"
+                  groupName="cardClusterCheckbox"
+                  [isChecked]="cardArray.buttonClicked === 'CardCluster' + (x + 1)"
+                  [showTitle]="true"
+                  [ariaLabel]="'Activate enter key to edit this user`s access'"
```

Comment:
instead of hard coding this into the design system, can this be included in the object that's passed into this component?

Reply:
Yeah, absolutely, definitely a bit of a lapse in judgement on my part

Reply:
I do think there is some merit to hardcoding the show title to true, as it seems like a universal accessibility need.